docs/custom.css

```diff
@@ -0,0 +1,3 @@
+.code-link::before {
+  content: "</>";
```

Comment:
This is a bit hacky.

We add the text via the `::before` element in order to prevent mkdocs from adding it to the table of contents as well.